source/_posts/2018-07-11-hassio-images.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Hass.io images"
+description: "We release our new images."
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 months we are happy to announce our new images for Hass.io, based on HassOS.
+
+Note that the descriptions of the new system can get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+We have build HassOS on top of the [Buildroot] framework. The focus of the system is to be a very small and high efficient operating system to run Docker like a hypervisor. It has installed just enough software to run a supervisor. We have also focussed on security, there are, for example, no default passwords and we use [AppArmor] to protect the applications and containers on HassOS.
```

Comment:
*We have build* => *We have built*

*and high efficient* => *and highly efficient*